features/liquidity/components/LiquidityBreakdown.tsx

```diff
@@ -73,13 +74,37 @@ const PoolHeader = ({ tokenA, tokenB, priceBreakdown }: PoolHeaderProps) => (
   </Inline>
 )
 
+const SwapFee = ({
+  protocolFee,
+  lpFee,
+}: {
+  protocolFee?: number
+  lpFee?: number
+}) => {
+  const protocolFeeSafe = protocolFee != undefined ? protocolFee : 0
```

Comment:
isnt this the same as setting `protocolFee = 0` and not making it optional?

Reply:
yes it is! I'm waiting on Ben for the `undefined` question though https://github.com/Wasmswap/wasmswap-interface/pull/260#discussion_r972535790. Since this would require reading the argument and we would have to remove the defaulting.